tools/c7n_azure/c7n_azure/resources/synapse.py

```diff
@@ -0,0 +1,12 @@
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('synapse')
+class Synapse(ArmResourceManager):
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Networking']
```

Comment:
this doesn't look like its networking related, its closer to analytics or database afaics